.storybook/customTheme.js

```diff
@@ -0,0 +1,32 @@
+import { create } from '@storybook/theming';
+import * as vars from '../src/styles/variables';
+
+export default create({
+  base: 'light',
+
+  colorPrimary: vars.bdlBoxBlue,
+  colorSecondary: vars.bdlBoxBlue,
+
+  // UI
+  appBg: vars.bdlGray05,
+  appContentBg: vars.white,
+  appBorderColor: vars.bdlGray10,
+  appBorderRadius: vars.bdlBorderRadiusSize,
+
+  // Typography
+  fontBase: 'Lato, "Helvetica Neue", Helvetica, Arial, sans-serif',
+  fontCode: 'monospace',
+
+  // Text colors
+  textColor: vars.bdlGray,
+  textInverseColor: vars.bdlGray02,
+
+  // Toolbar default and active colors
+  barTextColor: vars.white,
+  barSelectedColor: vars.white,
+  barBg: vars.bdlBoxBlue,
+
+  brandTitle: 'Box Elements',
+  brandUrl: 'https://opensource.box.com/box-ui-elements/',
```

Comment:
Should this point to the Storybook docs URL, i.e., 'https://opensource.box.com/box-ui-elements/storybook'?